libs/server/auth/api-rest/src/lib/controllers/auth.controller.ts

```diff
@@ -0,0 +1,27 @@
+import { Body, Controller, HttpException, HttpStatus, Post } from '@nestjs/common';
+import { AccessTokenBodyDto } from '../dto/access-token-body.dto';
+import { AuthResponse, Platform } from '@pimp-my-pr/shared/domain';
+import { AuthFacade } from '@pimp-my-pr/server/auth/core/application-services';
+
+@Controller('auth')
+export class AuthController {
```

Comment:
Add also swagger decorators like in the other controllers and in models that can be returned to the fronted. Checkout in Reademe.md how to check changes in swagger.